helmctlr/helm.go

```diff
@@ -89,10 +90,13 @@ func (c Controller) ResourceAdded(r *unstructured.Unstructured) {
 	if err := c.installOrUpdate(r); err != nil {
```

Comment:
I was expecting to see guards here based on reading the current status and/or checking for spec changes to make sure we don't process the same CR multiple times for no reason?

Reply:
Doesn't seem necessary after deploying this to a cluster. We can add it in a follow-up PR if needed.